Buildings/Fluid/AirFilters/BaseClasses/FiltrationEfficiency.mo

```diff
@@ -0,0 +1,89 @@
+within Buildings.Fluid.AirFilters.BaseClasses;
+model FiltrationEfficiency
+  "Component that calculates the filtration efficiency"
+
+  parameter Real mCon_nominal(
+    final unit = "kg")
+    "Maximum mass of the contaminant that can be captured by the filter";
+  parameter String substanceName[:] = {"CO2"}
+    "Name of trace substance";
+  parameter
+    Buildings.Fluid.AirFilters.BaseClasses.Characteristics.FiltrationEfficiencyParameters
+    filEffPar
+    "Filtration efficiency versus relative mass of the contaminant";
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput mCon(
+    final unit="kg")
+    "Mass of the contaminant captured by the filter"
+    annotation (Placement(transformation(extent={{-140,-20},{-100,20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput y[nConSub](
+    each final unit="1",
+    each final min=0,
+    each final max=1)
+    "Filtration efficiency of each contaminant"
+    annotation (Placement(transformation(extent={{100,-80},{140,-40}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealOutput rat(
+    final unit="1",
```

Comment:
In this class, the `rat` is a scale variable, while in class `Buildings.Fluid.AirFilters.BaseClasses.Characteristics.FiltrationEfficiencyParameters`, the `rat` is a vector. Are they same of different?
Through commit [e746684](https://github.com/lbl-srg/modelica-buildings/pull/3676/commits/e74668435a771556c67b852a56248f6defe49d3f), I removed `each` declaration. Is it a correct change? As in Modelica, `each` should only be used by the vector variable declaration.

Reply:
<code>Buildings.Fluid.AirFilters.BaseClasses.FilterationEfficiency</code> defines <code>rat</code> while in <code>Buildings.Fluid.AirFilters.BaseClasses.Characteristics.FiltrationEfficiencyParameters</code>, we declare a list of <code>rat</code> and each <code>rat</code> corresponds to a specific type of containment. Should I rename the <code>rat</code> in <code>Buildings.Fluid.AirFilters.BaseClasses.Characteristics.FiltrationEfficiencyParameters</code> to avoid confusions?